### PR TITLE
 report helpful error message for missing sensitive properties file

### PR DESCRIPTION
**Pull request by kunwarj**

### CODE REVIEW

File: nunaliit2-couch-command/src/main/java/ca/carleton/gcrc/couch/command/AtlasProperties.java [hunk 1]
```diff
@@ -102,52 +102,25 @@ static public AtlasProperties fromProperties(Properties props) throws Exception
...
+		FileInputStream fis = null;
+		try {
+			fis = new FileInputStream(configFile);
+			InputStreamReader reader = new InputStreamReader(fis,"UTF-8");
+			props.load(reader);
+		} catch(Exception e) {
+			throw new Exception("Unable to read config properties from: " + configFile.getAbsolutePath(), e);
+		} finally {
+			if( null != fis ){
+				fis.close();
```
Comment (alexgao1) [inline nunaliit2-couch-command/src/main/java/ca/carleton/gcrc/couch/command/AtlasProperties.java:123]: The old code did try catch when trying to close the file input stream, probably want to keep that here as well.

**Comment by alexgao1** [comment]
closes #179

**Review by alexgao1 — APPROVED** [review]

**Comment by kunwarj** [comment]
https://github.com/GCRC/nunaliit/issues/980

**Comment by alexgao1** [comment]
To note: the `sensitive.properties` file can no longer be generated with `nunaliit config` in this PR so far. However, this file should be able to be generated if it is missing.

**Comment by alexgao1** [comment]
(When missing a `sensitive.properties` file)
If I try to `update` or `run` now, the code throws a FileNotFoundException in addition to all the other messages.  
Even if I try to `config` with a missing `sensitive.properties` file, I'll also get the FileNotFoundException.

**Review by alexgao1 — APPROVED** [review]

**Pull request merged**